### PR TITLE
nlreturn: add block-size option

### PR DESCRIPTION
**Pull request by butuzov**
Adds configuration `block-size` that defines the size of the block (including return statement that is still "OK") so no return split is required.

Example

```yaml
linters-settings:
  nlreturn:
    block-size: 2
```

### CODE REVIEW

**Comment by butuzov** [comment]
Currently blocked as v2.2.0 has and off-by-one error that brings  `panic`

File: .golangci.example.yml [hunk 1]
```diff
@@ -158,6 +158,11 @@ linters-settings:
     # Exclude godoc examples from forbidigo checks.  Default is true.
     exclude_godoc_examples: false
 
+  nlreturn:
```
Comment (ldez) [inline .golangci.example.yml:161]: can move this section in this alphabetical place?

**Review by ldez — CHANGES_REQUESTED** [review]

File: .golangci.example.yml (hunk 1 above)
Comment (butuzov) [reply .golangci.example.yml]: sure. done.

**Comment by butuzov** [comment]
@ldez it's blocked because the current version that supports block-size also have off-by-one error and panics. I am waiting for @ssgreg to merge my PR

**Comment by butuzov** [comment]
@ldez maybe its worth make abc order for other parts too (linters settings, linters list in the manager) as another PR? It's so easy to miss (as I did)

**Review by ssgreg — APPROVED** [review]

**Comment by butuzov** [comment]
@ssgreg awesome! can you release `v2.2.1`?

**Review by ldez — APPROVED** [review]

**Pull request merged**